### PR TITLE
Put env var name for trust token in MagicStrings

### PR DESCRIPTION
**Pull request by rmunn**
Now we have a single place to change it when we need to.

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/lfmerge/139)

### CODE REVIEW

File: src/LfMerge.Core/MagicStrings.cs [hunk 1]
```diff
@@ -94,6 +94,7 @@ static MagicStrings()
 
 		// Other magic strings that don't fall into any particular category
 		public const string UnknownString = "***";
+		public const string EnvVar_TrustToken = "LD_TRUST_TOKEN";
```
Comment (rmunn) [inline src/LfMerge.Core/MagicStrings.cs:97]: Needs to become `LANGUAGE_DEPOT_TRUST_TOKEN`.

**Review by rmunn — COMMENTED** [review]
Changes needed before this can be merged.

File: src/LfMerge.Core/MagicStrings.cs (hunk 1 above)
Comment (megahirt) [inline src/LfMerge.Core/MagicStrings.cs:97]: Per Robin's comment above:
```suggestion
		public const string EnvVar_TrustToken = "LANGUAGE_DEPOT_TRUST_TOKEN";
```

Comment (megahirt) [reply src/LfMerge.Core/MagicStrings.cs:97]: I added a code suggestion to this effect

**Review by megahirt — APPROVED** [review]
LGTM.  Robin, what changes need to happen on K8s before we can deploy a new LFMerge with the new token name?

**Comment by megahirt** [comment]
> LGTM. Robin, what changes need to happen on K8s before we can deploy a new LFMerge with the new token name?

Actually, now that I've merged master into this branch, it looks like no changes are made since the trust token name was already changed.  This PR was simply outdated.

**Review by megahirt — APPROVED** [review]
:lgtm:

> Reviewed 1 of 3 files at r1, 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/lfmerge/139#-:-N9tjx5u7GjD8La9RiRy:bnfp4nl)* status: :shipit: complete! 1 of 1 LGTMs obtained (waiting on @rmunn)

**Pull request merged**